src/Raven.Server/Config/Categories/IndexingConfiguration.cs

```diff
@@ -614,6 +614,12 @@ public virtual bool RunInMemory
         [ConfigurationEntry("Indexing.Corax.VectorSearch.DefaultNumberOfCandidatesForQuerying", ConfigurationEntryScope.ServerWideOrPerDatabaseOrPerIndex)]
         public int CoraxVectorDefaultNumberOfCandidatesForQuerying { get; protected set; }
         
+        [Description("Order by score automatically when vector.search is inside query.")]
+        [DefaultValue(true)]
+        [IndexUpdateType(IndexUpdateType.Refresh)]
+        [ConfigurationEntry("Indexing.Corax.VectorSearch.OrderByScoreAutomatically", ConfigurationEntryScope.ServerWideOrPerDatabaseOrPerIndex)]
+        public bool CoraxVectorSearchOrderByScoreAutomatically { get; set; }
```

Comment:
why is this configurable? when would we want or recommend to set it to `false`?

Reply:
I would join that question - looks odd

Reply:
Is that an actual scenario - where you need to do that _globally_  for the index?

Reply:
Remember that this is a completely new feature - so no existing expectations.

When do I _not_ want to do that, *and* I didn't specify my own `order by` ?